manifests/upstream/sealed-secrets/base/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources: 
+- https://github.com/bitnami-labs/sealed-secrets/releases/download/v0.19.5/controller.yaml
```

Comment:
this will additionally need an 'images' directive to insert templating directives to pass version info through from the karina file.  Also, if the deployment features an emptyDir{} volume mount for things like logs you'll want to patch in the safe-to-evict annotation so that it doesn't break autoscaling.

once you've done that, you can try:
```
cd manifests
make sealed-secrets.yaml
```
and check that the manifest has been generated and the templates were inserted., then add the updated manifests to the PR